src/main/java/tanukidecor/block/RotatingTallBlock.java

```diff
@@ -174,11 +186,24 @@ public boolean canSurvive(BlockState pState, LevelReader pLevel, BlockPos pPos)
     }
 
     @Override
-    public void playerWillDestroy(Level pLevel, BlockPos pPos, BlockState pState, Player pPlayer) {
+    public BlockState playerWillDestroy(Level pLevel, BlockPos pPos, BlockState pState, Player pPlayer) {
         if (!pLevel.isClientSide && pPlayer.isCreative()) {
-            DoublePlantBlock.preventCreativeDropFromBottomPart(pLevel, pPos, pState, pPlayer);
```

Comment:
Why was this removed? The method in DoublePlantBlock still exists and can be access-widened, and I would prefer not to copy-pasta